roles/ceph-osd/tasks/docker/start_docker_osd.yml

```diff
@@ -117,18 +116,3 @@
     - ansible_os_family != 'CoreOS'
     - not osd_containerized_deployment_with_kv
 
```

Comment:
Here you're completely losing the ability to ron on !redhat !coreos.  Is that intentional?